platforms/mixin-lifecycle/build.gradle.kts

```diff
@@ -29,13 +34,14 @@ tasks {
     compileJava {
         options.release.set(17)
     }
-    
+
     remapJar {
         inputFile.set(shadowJar.get().archiveFile)
     }
 }
 
 architectury {
-    common("fabric", "quilt")
+//    common("fabric", "quilt")
```

Comment:
Remove commented out code

Reply:
this is to make it easy to reenable quilt

Reply:
it's already easy to re-enable, so it should be removed imo

commented out code is a codesmell